pilot/pkg/model/jwks_resolver.go

```diff
@@ -341,6 +341,7 @@ func (r *JwksResolver) refresher() {
 			r.refresh()
 		case <-closeChan:
 			r.refreshTicker.Stop()
+			return
```

Comment:
currently when we close the channel we are still looping here forever. should exit instead